web/src/beta/features/Editor/index.tsx

```diff
@@ -35,8 +35,7 @@ const Editor: React.FC<Props> = ({ sceneId, projectId, workspaceId, tab }) => {
               direction="vertical"
               gutter="end"
               size={metrics.propertyMenuMinWidth}
-              minSize={metrics.propertyMenuMinWidth}
-              maxSize={metrics.propertyMenuMaxWidth}>
```

Comment:
I guess you can delete `metrics.propertyMenuMaxWidth` from the `metrics` object

Reply:
They removed now.

Reply:
Sorry I meant you can delete [this line](https://github.com/reearth/reearth/blob/main/web/src/services/theme/reearthTheme/common/metrics.ts#L3).

So it's great to keep having `minSize` in this props and delete the above line.

Reply:
done - instead I make it propertyMenuWidth to set the initial size.